src/test/librbd/test_librbd.cc

```diff
@@ -2964,6 +2964,35 @@ TEST_F(TestLibRBD, TestIOToSnapshot)
   rados_ioctx_destroy(ioctx);
 }
 
+TEST_F(TestLibRBD, TestSnapshotDeletedIo)
+{
+  rados_ioctx_t ioctx;
+  rados_ioctx_create(_cluster, m_pool_name.c_str(), &ioctx);
```

Comment:
This `ioctx` is being leaked since `rados_ioctx_destroy` is not invoked at the end of the test.

Reply:
Right, and add rados_ioctx_destroy in the end.